.readthedocs.yml

```diff
@@ -12,3 +12,18 @@ python:
   version: 3.8
   install:
     - requirements: docs-requirements.txt
+    # Need to install the api/sdk in the venv for autodoc. Modifying sys.path
+    # doesn't work for pkg_resources.
+    # Keep these packages in sync with tox.ini.
+    - method: pip
```

Comment:
If these are installing from PyPI, then the contrib read the docs builds could fail if they depend on something unpublished. The "latest" tag on ReadTheDocs builds from the main branch on each commit

Reply:
That makes sense, then it sounds like we just need to put the git urls from before into this file.

Reply:
@aabmass I've updated with the [pip VCS install commands](https://pip.pypa.io/en/stable/cli/pip_install/#git), please give it a review when you have a chance?